react-native-libraries.json

```
@@ -13107,5 +13107,12 @@
     "android": true,
     "newArchitecture": true,
     "newArchitectureNote": "Library is a pure C++ TurboModule and requires the new architecture to be enabled. Doesn't work on legacy architecture"
+  },
+  {
+    "githubUrl": "https://github.com/jhugman/uniffi-bindgen-react-native",
+    "dev": true,
+    "ios": true,
+    "android": "true",
```

Hey @kraenhansen, thank you for adding a new entry! 👍 

To fix the CI you will need to replace the string value in `android` key with boolean:

```suggestion
    "android": true,
```

---

🤦 oh my